### PR TITLE
fix sysusers in noninteractive case

### PR DESCRIPTION
**Pull request by Mic92**

### CODE REVIEW

File: nix/noninteractive.nix [hunk 1]
```diff
@@ -30,6 +30,14 @@
   # prevents strace
   environment.defaultPackages = lib.mkForce [ pkgs.rsync pkgs.parted pkgs.gptfdisk ];
 
+  # normal users are not allowd with sys-users https://github.com/NixOS/nixpkgs/pull
```
Comment (flokli) [inline nix/noninteractive.nix:33]:
```suggestion
  # normal users are not allowed with sys-users .
  # see https://github.com/NixOS/nixpkgs/pull/328926
```

**Review by flokli — CHANGES_REQUESTED** [review]

File: nix/noninteractive.nix (hunk 1 above)
Comment (Mic92) [reply nix/noninteractive.nix]: Applied.

Comment (Mic92) [reply nix/noninteractive.nix]: Done.

File: nix/noninteractive.nix [hunk 2]
```diff
@@ -30,6 +30,15 @@
   # prevents strace
   environment.defaultPackages = lib.mkForce [ pkgs.rsync pkgs.parted pkgs.gptfdisk ];
 
+  # normal users are not allowed with sys-users
+  # see https://github.com/NixOS/nixpkgs/pull/328926
+  users.users.nixos = {
+    isSystemUser = true;
+    isNormalUser = lib.mkForce false;
+  };
+  users.users.nixos.group = "nixos";
```
Comment (nikstur) [inline nix/noninteractive.nix:39]: nit: 

```suggestion
  users.users.nixos = {
    isSystemUser = true;
    isNormalUser = lib.mkForce false;
    group = "nixos";
  };
```

**Review by nikstur — APPROVED** [review]

**Comment by evrim** [comment]
Rationale: see https://github.com/NixOS/nixpkgs/pull/328926

**Pull request merged**